Header header
float32[] dist_ave
float32[] angle_ave
int32[] angle_span

